packages/grafana-flamegraph/src/FlameGraphContainer.tsx

```diff
@@ -156,6 +168,8 @@ const FlameGraphContainer = ({
         colorScheme={colorScheme}
         onColorSchemeChange={setColorScheme}
         stickyHeader={Boolean(stickyHeader)}
+        stickyHeaderOffset={stickyHeaderOffset}
+        stickyHeaderZindex={stickyHeaderZindex}
```

Comment:
Is the zindex change needed? I would thought the offset should prevent any overlap anyway.

Reply:
that's a good point, i looked at and it's not needed so we can remove it 👍  i'm going to look into other ways to make ExploreToolbar stick to the top without making it sticky, based on @ifrost's comments

Reply:
i've looked into this and found a better solution that doesn't require any changes in flame graph